.github/workflows/continous-integration-docker.yml

```diff
@@ -205,8 +208,10 @@ jobs:
       height_grep_text: 'flushing database to disk height=Height'
 
   # Test that Zebra syncs and fully validates a few thousand blocks from a cached mandatory checkpoint disk
+  #
+  # If the state version has changed, waits for the new cached state to be created.
   test-stateful-sync:
-    needs: build
+    needs: [build, regenerate-stateful-disks]
```

Comment:
I think the condition I want here is:
- build has succeeded
- regenerate has succeeded or skipped

(but I think the default is to require both to succeed)

Reply:
This was fixed in commit 53989f9